InfiniteRecharge/src/main/java/frc/robot/config/Config.java

```diff
@@ -240,23 +231,11 @@ public void initMotorConfigArrays() {
         }
     }
 
-    public static class AutoConfig {
-        public double cruiseSpeed_mps = 0.3*14;
-        public double maxAcceleration_mps = 2;
-
-        public double b = 2;
-        public double zeta = 0.35;
-
-        public double kP = 0.911;
-    }
-
     public ShooterConfig shooter = new ShooterConfig();
     public BallManagementConfig ballManagement = new BallManagementConfig();
     public DriveConfig drive = new DriveConfig();
-    public IntakeConfig intake = new IntakeConfig();

```

Comment:
We still want the IntakeConfig and AutoConfig member variables to be part of the config class. We just want the class definitions to be regular class files instead of static inner classes like they are now. See the PR Elias raised for ShooterConfig:

https://github.com/BitBucketsFRC4183/FRC2020_Infinite_Recharge/pull/284